src/Diagnostics.RuntimeHost/Startup.cs

```diff
@@ -194,10 +194,13 @@ public void ConfigureServices(IServiceCollection services)
                 loggingConfig.ClearProviders();
                 loggingConfig.AddConfiguration(Configuration.GetSection("Logging"));
                 loggingConfig.AddDebug();
-                loggingConfig.AddEventSourceLogger();
```

Comment:
Do not remove AddEventSourceLogger() call. This is not event viewer logging. This is [logging for ETW events](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-3.1#event-source-provider). If we remove this, we will not get ETW emitted by ASP.NET Core. You can safely remove the AddEventLog() call which logs by default to EventLog.

Once we upgrade to .net 3.0 or above, we will need ETW to diagnose production issues via Profilers so lets keep the AddEventSourceLogger method.